README.md

```diff
@@ -221,15 +215,15 @@ To learn more about the Nightscout API, visit https://YOUR-SITE.com/api-docs.htm
 
 ### Required
 
-  * `MONGO_CONNECTION` - Your mongo uri, for example: `mongodb://sally:sallypass@ds099999.mongolab.com:99999/nightscout`
```

Comment:
did you delete MONGO_CONNECTION on purpose?

Reply:
Far as I could tell, `MONGO_CONNECTION` has been deprecated; we use `MONGODB_URI` now. No?

Reply:
MONGODB_URI should be listed here. Also the MONGODB_COLLECTION is not a required variable?

Reply:
It is required, per the Heroku template.

Reply:
Added MONGODB_URI to the Required section.